packages/terra-outline-view-table/tests/jest/Table.test.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import OutlineTableView from '../../src/Table';
+import OutlineTableView from '../../src/OutlineViewTable';
```

Comment:
Should this also be `OutlineViewTable`? Same for [here](https://github.com/cerner/terra-core/blob/aa978cd60fd0dfabac3047e2adf844b41eaf12ae/packages/terra-outline-view-table/src/terra-dev-site/doc/outline-view-table/example/DefaultTable.jsx#L2) and [here](https://github.com/cerner/terra-core/blob/aa978cd60fd0dfabac3047e2adf844b41eaf12ae/packages/terra-outline-view-table/src/terra-dev-site/test/outline-view-table/OutlineViewTable.test.jsx#L2).
```suggestion
import OutlineViewTable from '../../src/OutlineViewTable';
```

Reply:
Good catch. I'll update.

Reply:
https://github.com/cerner/terra-core/pull/3692/commits/5d5e261a193d64dec07f1f49f29f8e2ee8baf2c7